.github/workflows/ci.yml

```diff
@@ -313,6 +313,32 @@ jobs:
             benchmark: 1
             setenvs: export PUGIXML_VERSION=v1.15
             optional_deps_append: "openjph;Qt6"
+          - desc: VFX2025 Debug gcc11/C++17 py3.11 exr3.3 ocio2.4
+            nametag: linux-vfx2025-debug
+            runner: ubuntu-latest
+            container: aswf/ci-oiio:2025
+            cxx_std: 17
+            build_type: Debug
+            python_ver: "3.11"
+            simd: "avx2,f16c"
+            fmt_ver: 11.2.0
+            pybind11_ver: v2.13.6
+            benchmark: 1
```

Comment:
Just double-checking that `benchmark` was intended to be enabled here on the Debug config?

Reply:
um, no, thanks for catching. I think that's a cut and paste error.

Reply:
and I found a couple others I can eliminate